### PR TITLE
South Africa_Sinombulelo_Pungula_Github_Lesson_1

### PR DESCRIPTION
**Pull request by sinopungula**
# Your Details

- Your Name: Sinombulelo
- Your City: Cape Town
- Your Slack Name: Sinopungula

# Homework Details

- Module: 1
- Week: 2

### CODE REVIEW

File: homework.md [hunk 1]
```diff
@@ -92,13 +93,13 @@ A place for us to store and share our code.
...
+Save your code onto a repository
 
 ## Question 16
 
 What program do we use to write code at CodeYourFuture?
 
-Microsoft Word
+Visual Studio Code
 
 ## Question 17
```
Comment (LawrenceBaatjies) [inline homework.md:104]: Hi Sinombulelo.
 Those are so pretty  Awesome detailed answers, however I've noticed that you haven't completed Question #17 and #18 You have not Inputted your Own Answers . You might want to take a look at it?

Comment (sinopungula) [reply homework.md:104]: Hi Lawrence!

I figured since the answers are correct, I'd leave them be but I will make them my own. Thanks!

**Review by BrettScheepers — COMMENTED** [review]
Answers look good

**Comment by sirpfaira** [comment]
You have done a good work.

File: homework.md [hunk 2]
```diff
@@ -92,22 +93,22 @@ A place for us to store and share our code.
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+Save your code onto a repository on Git
```
Comment (Ntshembo-Hlongwane1) [inline homework.md:96]: > Save your code onto a repository on Git
>
So this what you have to understand that **Git** is **tool** that you make use of to save changes to your local and remote repository plus many more other tasks... 

### "Committing is saving all your file changes to your **local repository** by using 'Git' "

**Pull request closed without merging**